manifests/postgresql.crd.yaml

```diff
@@ -574,6 +574,12 @@ spec:
                         - PreferNoSchedule
                     tolerationSeconds:
                       type: integer
+              topologySpreadConstraints:
+                type: array
+                nullable: true
+                items:
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
Can we not add all the fields of a topologySpreadConstraint like with what we have for `nodeAffinity`. I feel, it's too lazy and unsafe to allow arbitrary fields with `x-kubernetes-preserve-unknown-fields: true`